code_search/docker/ks/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM debian:7
+FROM google/cloud-sdk:225.0.0-slim
```

Comment:
Why do we need gcloud?

Reply:
revert. it was used for copying vocabulary file